src/plugins/laya-la-scmc/edit.vue

```diff
@@ -78,6 +78,23 @@ Dependencies:
               {{ y18n('layaLaScmc.edit.mc') }}
             </label>
           </div>
+          <div class="form-check form-check-inline align-text-top">
+            <input
+              id="scmc-tf"
+              v-model="variation"
+              class="form-check-input"
+              type="radio"
+              name="tf"
+              :value="tf"
+              @click="$bvModal.show('confirm-change-tf')"
```

Comment:
Modal shows, but if you click "cancel", the variation is still changed and you have uneditable input fields